Sources/KeyboardSupport/KeyboardScrollable.swift

```diff
@@ -58,35 +58,37 @@ public struct KeyboardInfo {
 /// Enables scrolling views to the first responder when a keyboard is shown. Must be used with a UIScrollView or one of its subclasses.
 public protocol KeyboardScrollable: class {
     var minimumPaddingAroundInput: UIEdgeInsets { get }
-    
+
     var keyboardScrollableScrollView: UIScrollView? { get }
     var keyboardWillShowObserver: NSObjectProtocol? { get set }
     var keyboardWillHideObserver: NSObjectProtocol? { get set }
-    
+
     var preservesContentInsetWhenKeyboardVisible: Bool { get }
-    
+
     /// Must be called during screen appearance ('viewWillAppear') to allow for keyboard notification observers to be registered.
     func setupKeyboardObservers()
-    
+
     /// Must be called during screen disappearance ('viewWillDisappear') to allow for keyboard notification observers to be unregistered.
     func removeKeyboardObservers()
-    
+
     /// Called when the keyboard is showing.
     func keyboardWillShow(keyboardInfo: KeyboardInfo)
-    
+
     /// Called when the keyboard is hiding.
     func keyboardWillHide(keyboardInfo: KeyboardInfo)
 }
 
 public extension KeyboardScrollable where Self: UIViewController {
-    
+
     // MARK: KeyboardScrollable Conformance
     var minimumPaddingAroundInput: UIEdgeInsets {
         return .zero
     }
-    
+
     var preservesContentInsetWhenKeyboardVisible: Bool { return true }
-    
+
+    private var isNotPopover: Bool { return popoverPresentationController == nil }
```

Comment:
Do you think we could remove the negative from the variable name? I think it reads a little cleaner without it.
```swift
private var isPopover: Bool { return popoverPresentationController != nil }
```

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/61/commits/e3ba7670babb46efdc74bbe985614efd1768432b